src/components/Profile/Landing/index.tsx

```diff
@@ -1,13 +1,184 @@
-import { FC } from 'react';
+import { FC, useEffect, useState } from 'react';
 import { useParams } from 'react-router-dom';
 import bg from 'assets/illustrations/profile/bg.svg';
 import NavBar from 'components/NavBar';
 import ProfileCard from 'components/ProfileCard';
+import { useDispatch } from 'react-redux';
+import Shepherd from 'shepherd.js';
+import { editProfile } from 'actions/profile';
 import styles from './index.module.scss';
 import ProfileContent from '../ProfileContent';
 import { useFetchPublicProfile } from './hooks';
 
 const Landing: FC = () => {
+  const [onboardStatus, setOnboardStatus] = useState(
+    localStorage.getItem('onboardStatus')
+  );
+  const dispatch = useDispatch();
+  const tour = new Shepherd.Tour({
+    useModalOverlay: true,
+    defaultStepOptions: {
+      classes: 'shadow-md bg-purple-dark',
+      scrollTo: true,
+    },
+  });
+
+  window.addEventListener('storage', () => {
+    setOnboardStatus(localStorage.getItem('onboardStatus'));
+  });
+  useEffect(() => {
+    if (onboardStatus === 'started') {
+      setTimeout(() => {
+        tour.show('step1');
+      }, 500);
+    }
+    if (onboardStatus === 'partial') {
+      tour.show('step3');
+      localStorage.removeItem('onboardStatus');
```

Comment:
There's a `removeItem` function in `localStorageFn.ts` file which can be used to remove the value